scss/elements/_sections.scss

```diff
@@ -168,6 +168,17 @@
 				margin-top: $baseline;
 			}
 
+			.cmd-heading {
+				padding-left: ($baseline * 6);
+				@include breakpoint(sm) { 
+					padding-left: ($baseline * 4);
+				}
+			}
+
+			.cmd-list {
+				padding: 0px;
+			}
+		
```

Comment:
I don't think we should use service/product specific terms (e.g. "cmd") in our code unless we really need to. And we should be able to achieve this styling by using `padding-left-sm--2 padding-left-md--3` and `padding-left--0`